### ISSUE TITLE
Add cake-contrib user

### ISSUE THREAD

**Issue by gep13** [asmagin/Cake.Sitecore#4]
First of all, I wanted to thank you for adding to the Cake community by adding this addin.

I was just wondering if you had seen this blog post:

http://cakebuild.net/blog/2016/08/cake-contribution-organization

We are currently going through a process of increasing the visibility of addins, and also trying to ensure their long term maintainability.

To that end, we are asking addin creators to add the cake-contrib user on NuGet as a co-owner (this can be done through the NuGet website by clicking on Manage Owners on the package page).

Would you be interested in doing this? If you have any questions about this, please let me know. There was some initial concern that the Cake Team were trying to "take over" packages, and that couldn't be further from the truth, and if you have this concern, or others, I would like to address them.

Thanks!

**Comment by asmagin** [comment]
Done

**Comment by asmagin** [comment]
I'm fine moving it, as soon as I still can have full control over the repo.

**Comment by gep13** [comment]
@asmagin absolutely, you would have the same admin level access to the repo as you currently have here under your own GitHub user.  Let me know if you would like to proceed with the transfer.

**Comment by gep13** [comment]
@asmagin just wanted to follow up with this again.  Is this something that you would like to proceed with?

**Comment by augustoproiete** [comment]
Hey @asmagin it's been a while, but are you still interested in moving this repo to the [Cake-Contrib](https://github.com/cake-contrib) organization?
